bodhi/tests/server/services/test_updates.py

```diff
@@ -3583,7 +3582,7 @@ def test_disabled_button_for_autopush(self, *args):
         resp = self.app.get(f"/updates/{resp.json['alias']}", headers={'Accept': 'text/html'})
         assert 'text/html' in resp.headers['Content-Type']
         assert nvr in resp
-        assert 'Stable by Karma' not in resp
+        assert 'Stable by Karma' in resp
```

Comment:
The changes in this file are wrong. Here the purpose was to test that on the update page the 'stable by karma' line was not showed when that function was disabled.
Now the line is always visible, so we need to change the logic of the test. For example we can test that `The update will not be automatically pushed to stable by karma` is in `resp` when autopush is disabled, while when enabled we can use `The update will be automatically pushed to stable  when karma reaches {update_info['stable_karma']}`